packages/vscode-extension/src/notifications/NotificationHandler.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Notification } from "@kogito-tooling/notifications/dist/core/index";
+
+interface NotificationHandler {
```

Comment:
```suggestion
export interface NotificationHandler {
```

You can export directly and remove line 25.

Reply:
Isn't this the name interface as `NotificationsApi`? Why do we need this one too? If they have different purposes, it's fine and it's not duplication, but if they're ALWAYS going to be the same, we can loose this one.